### PR TITLE
initialization

### PR DESCRIPTION
**Pull request by jakai7**
In this PR I have tried to solve some issues regarding the initialization of the App:

- Metamask Extension Initialization
- Polkadotjs Extension Initialization
- Fetching account balances after initialization

### CODE REVIEW

**Comment by musnit** [comment]
I'm getting weird/inconsistent UI when i load https://deploy-preview-5--polkaeth-demo.netlify.app/

sometimes as below with the blue loading bar

image

if i just refresh, i get Please install the Polkadotjs Browser Extension!

this is after i have previously accepted access for both polkadotjs and metamask.

File: src/AppETH.tsx [hunk 1]
```diff
@@ -52,6 +56,19 @@ function AppETH ({ contract, defaultAccount, web3 }: Props): React.ReactElement<
   const [polkadotRecipient, setPolkadotRecipient] = useState(String);
   const [depositAmount, setDepositAmount] = useState(String);
 
+  const [polkaEthBalance, setPolkaEthBalance] = useState(String);
+
+  if(defaultAccount === undefined) {
+    alert("Please install MetaMask to use this application!");
```
Comment (musnit) [inline src/AppETH.tsx:62]: plz dont use alerts - the div below is fine

Comment (musnit) [reply src/AppETH.tsx:62]: this is in appeth and app? do we need it in both?

File: src/AppETH.tsx [hunk 2]
```diff
@@ -52,6 +56,19 @@ function AppETH ({ contract, defaultAccount, web3 }: Props): React.ReactElement<
...
+    return(<div><p>Please install MetaMask to use this application!</p></div>)
```
Comment (musnit) [inline src/AppETH.tsx:63]: we should probably have a button here to try connect to metamask again if first time fails or something

File: src/App.tsx [hunk 3]
```diff
@@ -26,11 +29,28 @@ function BridgeApp () {
...
+    polkadotExtension();
+  });
+
   if (!ethEnabled()) {
     alert("Please install MetaMask to use this application!");
     return(<div><p>Please install MetaMask to use this application!</p></div>)
   }
 
+  if(!polkaEthEnabled) {
+    return(<div><p>Please install the Polkadotjs Browser Extension!</p></div>)
```
Comment (musnit) [inline src/App.tsx:51]: we should probably have a button here to try connect to polkadotjs again if first time fails or something

File: src/AppETH.tsx (hunk 1 above)
Comment (jakai7) [reply src/AppETH.tsx:62]: Alright I will remove that.

File: src/App.tsx (hunk 3 above)
Comment (jakai7) [reply src/App.tsx:51]: Good idea.

**Comment by jakai7** [comment]
@musnit I realized these commits are not really clean so I reseted the branch, will push new commits and open a new PR.

**Pull request closed without merging**